pwiz_tools/Skyline/Controls/Graphs/DetectionPlotData.cs

```diff
@@ -27,7 +27,9 @@
 using JetBrains.Annotations;
 using pwiz.Common.Collections;
 using pwiz.Common.SystemUtil;
-using pwiz.Skyline.Util;
+using pwiz.Skyline.Util;
```

Comment:
The extra two empty lines seem unnecessary. Please review your diffs more carefully.